resources/user-agents/browsers/firefox/firefox-3-7.json

```diff
@@ -23,7 +23,7 @@
         "Cookies": "true",
         "JavaScript": "true",
         "CssVersion": "3",
-        "Beta": "true"
+        "Alpha": "true"
```

Comment:
I switched this one to Alpha, since everything I could find showed that 3.7 was an alpha release before they decided to change the version to 4.0.

https://en.wikipedia.org/wiki/Firefox_version_history#Release_history (Firefox 4.0 section of that table).